lib/pages/chat_page.dart

```diff
@@ -226,4 +244,92 @@ class _ChatPageState extends State<ChatPage> {
       _textController.clear();
     }
   }
+
+  void showFastCall(BuildContext context) {
+    List<Map<String, Object>> fastCallInfo = [
+      {
+        "MedicalTeamName": "อ.ฉัตรกมล ประจวบลาภ",
+        "PhoneNumber": "086-357-3877",
+      },
+      {
+        "MedicalTeamName": "อ.เอมปภา ปรีชาธีรศาสตร์",
+        "PhoneNumber": "062-926-2253",
+      },
+      {
+        "MedicalTeamName": "อ.พรนภา นาคโนนหัน",
+        "PhoneNumber": "089-493-3902",
+      },
+      {
+        "MedicalTeamName": "ผศ.ดร. รสสุคนธ์ วาริทสกุล",
+        "PhoneNumber": "081-642-3478",
+      },
+    ];
```

Comment:
Can we move this to a new flie as a constant.

Reply:
28f70eb Fixed ka